src/latticpp/ckks/keygen.cpp

```diff
@@ -139,8 +143,22 @@ namespace latticpp {
                                          rotKeys.getRawHandle());
     }
 
-    BootstrappingKey genBootstrappingKey(const KeyGenerator &keygen, const Parameters &params, const BootstrappingParameters &bootParams, const SecretKey &sk, const RelinearizationKey &relinKey, const RotationKeys &rotKeys) {
-        return BootstrappingKey(lattigo_genBootstrappingKey(keygen.getRawHandle(), params.getRawHandle(), bootParams.getRawHandle(), sk.getRawHandle(), relinKey.getRawHandle(), rotKeys.getRawHandle()));
+    // DEPRACATED
+    BootstrappingKey genBootstrappingKey(const KeyGenerator &keygen, const Parameters &params, const BootstrappingParameters &bootParams, const SecretKey &sk, const RelinearizationKey & /*relinKey*/, const RotationKeys &/*rotKeys*/) {
+        return genBootstrappingKey(keygen, params, bootParams, sk);
+    }
```

Comment:
I'm not terribly concerned with backwards compatibility; I suggest we just kill this function.
```suggestion
```